services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -3845,8 +3856,11 @@ private void truncateTopicsOptionally(
       boolean incPushEnabledBatchPushSuccess = !incrementalPushVersion.isPresent() && store.isIncrementalPushEnabled();
```

Comment:
GH does not allow me to post comment on code far way from changed lines :)  can you  please update the comments before this to indicate clearly when we can truncate the parent topic/